Sources/Bugsnag/Configuration.swift

```diff
@@ -74,7 +74,17 @@ public struct Configuration: ConfigurationType {
         field: Field,
         config: Config
     ) throws -> [String]? {
-        return config[field.rawValue]?.array as? [String]
+        // Get array
+        guard let platforms = config[field.rawValue]?.array else {
```

Comment:
Consider renaming this constant to something more generic - maybe just `array` or `configArray`?